.github/workflows/_coverage.yml

```diff
@@ -1,135 +0,0 @@
-name: xla-test-coverage
-on:
-  workflow_call:
-    inputs:
-      docker-image:
-        required: true
-        type: string
-        description: Image to test on
-      runner:
-        required: false
-        type: string
-        description: Runner type for the test
-        default: linux.12xlarge
-      collect-coverage:
-        required: false
-        type: boolean
-        description: Set to true to collect coverage information
-        default: false
-      timeout-minutes:
-        required: false
-        type: number
-        default: 270
-        description: |
-          Set the maximum (in minutes) how long the workflow should take to finish
-      disable-pjrt:
-        required: false
-        type: string
-        default: 0
-        description: Whether to disable PJRT tests
-      disable-xrt:
-        required: false
-        type: string
-        default: 0
-        description: Whether to disable XRT tests
-      test-script:
-        required: false
-        type: string
-        default: test.sh
-        description: Which test script to run
-
-    secrets:
-      gcloud-service-key:
-        required: true
-        description: Secret to access Bazel build cache
-jobs:
-  test:
-    runs-on: ${{ inputs.runner }}
-    timeout-minutes: ${{ inputs.timeout-minutes }}
-    env:
-      DOCKER_IMAGE: ${{ inputs.docker-image }}
-      WORKDIR: /var/lib/jenkins/workspace
-      GCLOUD_SERVICE_KEY: ${{ secrets.gcloud-service-key }}
-      USE_COVERAGE: ${{ inputs.collect-coverage && '1' || '0' }}
-      XLA_SKIP_XRT_TESTS: ${{ inputs.disable-xrt }}
```

Comment:
Does the new parallelized execution model skip jobs the same way as before @yeounoh?